CHANGELOG.md

```diff
@@ -11,6 +11,19 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [2.1.17]
+
+### Added
+
+- Added `layers.SSRU`, which implements a Simpler Simple Recurrent Unit as described by Kim et al, 2019.
```

Comment:
Maybe add the full paper title here and/or in the SSRU class?

Reply:
Sure.